qcodes/tests/test_json.py

```diff
@@ -42,7 +48,8 @@ def test_numpy_good(self):
             'length': 45.23,
             'points': [12, 24, 48],
             'RapunzelNumber': {'__dtype__': 'complex', 're': 4.89, 'im': 0.11},
-            'verisimilitude': {'__dtype__': 'complex', 're': 0, 'im': 1}
+            'verisimilitude': {'__dtype__': 'complex', 're': 0, 'im': 1},
+            'myuserdict': {'a': 1}
```

Comment:
Prior to this fix that would have been a string as in  `'myuserdict': "{'a': 1}"`